### PR TITLE
BITAU-96 Setup AIDL interface and files

### PR DESCRIPTION
**Pull request by ahaisting-livefront**
## 🎟️ Tracking

https://livefront.atlassian.net/browse/BITAU-96

## 📔 Objective

The goal here is to setup the main `BridgeService` aidl interface and corresponding models. 

## 📸 Screenshots

N/A

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
  issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

**Comment by ahaisting-livefront** [comment]
Thanks for the comments, @SaintPatrck!! 🤗 

Especially the typos... apparently spellcheck isn't enabled for `.aidl` files.

File: bridge/src/main/aidl/com/bitwarden/bridge/IBridgeService.aidl [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    // ==============
+    // Add TOTP Item
+    // ==============
+
+    // Returns an intent that can be launched to navigate the user to the add Totp item flow
+    // in the main password manager app.
+    Intent createAddTotpLoginItemIntent();
+
+    void setPendingAddTotpLoginItemData(in EncryptedAddTotpLoginItemData data);
```
[the remark this answers is not in the document]
Comment (ahaisting-livefront) [reply bridge/src/main/aidl/com/bitwarden/bridge/IBridgeService.aidl:59]: Good callout!

35d9c3fb428afbdffa9c5a9b8858feb7f278831e

**Review by SaintPatrck — APPROVED** [review]

**Comment by dseverns-livefront** [comment]
AIDL black magic, nice

**Comment by ahaisting-livefront** [comment]
Thanks, @dseverns-livefront !

**Pull request merged**